src/_P035_IRTX.ino

```diff
@@ -164,18 +164,18 @@ boolean Plugin_035(byte function, struct EventStruct *event, String &command)
   return success;
 } // Plugin_035 END
 
-boolean handleIRremote(const String &cmd) {
+bool handleIRremote(const String &cmd) {
 
-  String IrType = "";
-  String TmpStr1 = "";
+  String IrType;
+  String TmpStr1;
 
   uint64_t IrCode = 0;
   uint16_t IrBits = 0;
   uint16_t IrRepeat = 0;
-  String ircodestr = "";
+  String ircodestr;
 
   StaticJsonDocument<200> docTemp;
-  DeserializationError error = deserializeJson(docTemp, cmd.substring(cmd.indexOf(',') + 1, cmd.length()));
+  DeserializationError error = deserializeJson(docTemp, parseStringToEnd(cmd, 2));
```

Comment:
cmd.length() is missing.

Reply:
But that's only needed for substring, right?
parseStringToEnd is doing all the magic for you, including stripping matching quotes if present.
It does also convert to lower case.